mongo-data-sync/load-secrets-and-run.sh

```diff
@@ -0,0 +1,12 @@
+#!/bin/sh
```

Comment:
```suggestion
#!/bin/bash
```

ditto https://github.com/artsy/docker-images/pull/113#discussion_r1914848970